cli/cmd/stat.go

```diff
@@ -363,6 +367,14 @@ func getSuccessRate(r pb.StatTable_PodGroup_Row) float64 {
 	return float64(success) / float64(success+failure)
 }
 
+func getPercentSecured(r pb.StatTable_PodGroup_Row) float64 {
+	reqTotal := r.Stats.SuccessCount + r.Stats.FailureCount
+	if r.Stats.SecuredRequestCount == 0 {
```

Comment:
should this be:
```go
if reqTotal == 0 {
```

Reply:
whoops, yes it should! thanks for catching!